docs/missing-segments-in-nested-resource-list-operation.md

```diff
@@ -0,0 +1,82 @@
+# MissingSegmentsInNestedResourceListOperation
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-11
+
+## Description
+
+A nested resource type's List operation must include all the parent segments in its api path.
+
+## How to fix
+
+To fix this, ensure that the list operation's api path for the nested resource includes all the parent segments.
+
+### Examples
+
+The below path is invalid because there is no Parent path definition for the Nested path defined.
+
+```json
+"paths": {
+// No Parent Resource Get Call defined
+"undefined parent path"
+
+//Nested Resource Type Get Call definition
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ProviderNamespace/parentResourceType/{parentInstance}/nestedResourceType": {
+    "get/put/delete/post": {
```

Comment:
>/put/delete/post [](http://example.com/codeflow?start=8&length=16)

The comment says its a get api but we have all verbs here. The comment and the verbs used need to be in sync

Reply:
pl check everywhere

Reply:
I see that your implementation is scoped to get, so just mention get here

Reply:
Replaced all verbs to just Get